### ISSUE TITLE
does the umbrel app store open source?

### ISSUE THREAD

**Issue by liesauer** [getumbrel/umbrel-community-app-store#6]
hello umbrel team, i am learning to write an entire app store, the design of [apps.umbrel.com](https://apps.umbrel.com/) is really shining for me, does the umbrel app store open source? so i can learn something from it, tks.

**Comment by chezinut** [comment]
@liesauer yes https://github.com/getumbrel/umbrel-apps bro this repo is template so you can make own app store not curated by umbrel
